test/torch_np/numpy_tests/core/test_multiarray.py

```diff
@@ -75,7 +75,11 @@
 IS_PYSTON = False
 HAS_REFCOUNT = True
 
-from numpy.core.tests._locales import CommaDecimalPointLocale
+if numpy.__version__ > "2":
+    # numpy 2.0 +, see https://numpy.org/doc/stable/release/2.0.0-notes.html#renamed-numpy-core-to-numpy-core
+    from numpy._core.tests._locales import CommaDecimalPointLocale
+else:
+    from numpy.core.tests._locales import CommaDecimalPointLocale
```

Comment:
@albanD who would be a good person to review numpy testing changes?

Reply:
Alban is on vacation. Maybe @ezyang knows?